cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -243,6 +243,8 @@ pub trait ClientTrait {
     ) -> Result<SearchResults, VersionsError>;
 }
 
+const PAGE_SIZE: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(10) };
+
 #[async_trait]
 impl ClientTrait for CatalogClient {
```

Comment:
🚧 There isn't any unit or integration test coverage for the real client methods in here. I see that progenitor can [generate httpmock helpers](https://docs.rs/progenitor/latest/progenitor/struct.Generator.html#method.httpmock), which might be useful for adding coverage in the future.